spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe PagesController, type: :controller do
+
+  describe "GET #about" do
+    it "returns http success" do
+      get :about
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #contact" do
+    it "returns http success" do
+      get :contact
+      expect(response).to have_http_status(:success)
+    end
+  end
+
```

Comment:
`Rubocop` is complaining because `Extra empty line detected at block body end.`, so `delete` that line and `rubocop` should be happy!